services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
The type hint `list[dict]` is correct for Python 3.9 and above. Ensure your environment supports this syntax. If not, consider using `List[Dict]` from the `typing` module for compatibility.